recipe/meta.yaml

```diff
@@ -0,0 +1,16 @@
+{% set data = load_setup_py_data() %}
```

Comment:
We should add a README.md file in `recipe/` briefly explanation the conda-build process.